imports/include/niScopePrivate.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
```

Comment:
Note: I stripped everything else out except the one method I needed to get things building and also not expose too much.